scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -163,7 +163,9 @@ final class Dialect private[meta] (
     // Are binary literals allowed? SIP-42.
     val allowBinaryLiterals: Boolean,
     // Are tracked parameters allowed? docs: https://dotty.epfl.ch/docs/reference/experimental/modularity.html
-    val allowTrackedParameters: Boolean
+    val allowTrackedParameters: Boolean,
+    // https://github.com/scala/improvement-proposals/pull/81/files
```

Comment:
should we use something more readable, like https://docs3.scala-lang.org/sips/sips/typeclasses-syntax.html?

Reply:
Changed to allowImprovedTypeclassSyntax

Reply:
sorry, I meant here not to change the name of the flag, but to include a different URL for reference :) profuse apologies.

the scala-lang doc is more readable than the textual pull request. the original flag name was absolutely fine, can be restored if you like.